script.js

```diff
@@ -0,0 +1,128 @@
+// Mobile menu popup
+
+const menu = document.querySelector('#mob-menu');
+const hamburger = document.querySelector('#hamburger-icon');
+const closebtn = document.querySelector('#x-iconid');
+const navlinks = document.querySelector('#navLinks');
+
+hamburger.addEventListener('click', () => {
+  menu.style.display = 'block';
+  hamburger.style.display = 'none';
+});
+
+closebtn.addEventListener('click', () => {
+  menu.style.display = 'none';
+  hamburger.style.display = 'block';
+});
+
+navlinks.addEventListener('click', () => {
+  menu.style.display = 'none';
+  hamburger.style.display = 'block';
+});
+
+// Featured Staff
+
+const speakers = [
+  {
+    image_url: './images/p1-yochai.png',
+    name: 'Yochai Benkler',
+    designation:
+      'Piolt',
+    introduction:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Corporis.',
+  },
+  {
+    image_url: './images/p2-kilnam.png',
+    name: 'Kilnam Chon',
+    designation: 'Senior Trainer',
+    introduction:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Corporis.',
+  },
+  {
+    image_url: './images/p3-sohyeong.png',
+    name: 'SohYeong Noh',
+    designation: 'Tour Guide',
+    introduction:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Corporis.',
+  },
+  {
+    image_url: './images/p4-julia.png',
+    name: 'Julia Leda',
+    designation: 'CEO of Administration Department',
+    introduction:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Corporis.',
+  },
+  {
+    image_url: './images/p5-lila.png',
+    name: 'Lila Tretikov',
+    designation: 'Manager',
+    introduction:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Corporis.',
+  },
+  {
+    image_url: './images/p6-ryan.png',
+    name: 'Ryan Merkley',
+    designation: 'Cordinator',
+    introduction:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Corporis.',
+  },
+];
+
+const featuredSpeakerSection = document.createElement('section');
+featuredSpeakerSection.id = 'featured-speakers';
+{
+  const featuredSpeakerHeading = document.createElement('h2');
+  featuredSpeakerHeading.innerText = 'Featured Speakers';
+  featuredSpeakerSection.appendChild(featuredSpeakerHeading);
+}
+{
+  const featuredSpeakerLineSeperator = document.createElement('hr');
+  featuredSpeakerSection.appendChild(featuredSpeakerLineSeperator);
+}
+{
+  const featuredSpeakerList = document.createElement('ul');
+  featuredSpeakerList.id = 'featured-speakers-list';
+  for (let i = 0; i < speakers.length; i += 1) {
+    const featuredSpeakerListItem = document.createElement('li');
+    {
+      const featuredSpeakerImage = document.createElement('img');
+      featuredSpeakerImage.alt = 'staff member';
```

Comment:
- [ ] Nice job on generating the speaker's section with javascript, however, I think it would be better to use the same layout for the original template on the desktop view. check this :point_down: 

The template | Your design 
----|---
![image](https://user-images.githubusercontent.com/65068771/182009557-34c8e3b6-233a-4e85-88fd-7abf46b3fc6d.png) | ![image](https://user-images.githubusercontent.com/65068771/182009580-13fd972c-f930-43b3-86f8-c2490c1324df.png)